src/sentry/utils/snuba.py

```diff
@@ -140,10 +144,30 @@ def query(start, end, groupby, conditions=None, filter_keys=None,
             raise SnubaError(body['error'])
         else:
             raise SnubaError('HTTP {}'.format(response.status))
+    return body
+
+
+def query(start, end, groupby, conditions=None, filter_keys=None,
+          aggregations=None, rollup=None, arrayjoin=None, limit=None, orderby=None,
+          having=None, referrer=None, is_grouprelease=False, selected_columns=None):
 
+    groupby = groupby or []
+    conditions = conditions or []
+    having = having or []
```

Comment:
some of these like `having` which aren't used anywhere in this function except to be passed down to `raw_query` don't need to be modified here.